improver/cli/cubelist_extract.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
-# (C) Crown copyright, Met Office. All rights reserved.
+# (C) Crown Copyright, Met Office. All rights reserved.
 #
-# This file is part of IMPROVER and is released under a BSD 3-Clause license.
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
```

Comment:
Should we use `"` here rather than `'`?

Reply:
I tend to use these interchangably so I'm not sure what the distinction would be. Regardless the changes I've made are just so that these files match all the other files in IMPROVER so we should just follow their lead for consistency across the repo.

Reply:
If it is consistent with the rest of the repo then choice is fine with me.